src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java

```diff
@@ -78,8 +78,8 @@ public EditPersonDescriptorBuilder withAddress(String address) {
     /**
      * Sets the {@code Tag} of the {@code EditPersonDescriptor} that we are building.
      */
-    public EditPersonDescriptorBuilder withTags(String tags) {
-        descriptor.setTag(new Tag(tags));
+    public EditPersonDescriptorBuilder withTags(String tag) {
+        descriptor.setTag(new Tag(tag));
```

Comment:
Reviewing this file has been enlightening, as I've gained valuable insights and knowledge from the debugging process. I truly appreciate your assistance in troubleshooting the edit test case jawaaaad. (chatgpt started and i ended)